config/rbac/leader_election_role.yaml

```diff
@@ -0,0 +1,27 @@
+# permissions to do leader election.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: leader-election-role
```

Comment:
Maybe need to change the name to something more specific?

Reply:
This file was a copy of internal/controller/config/rbac/kube_api_roles.yaml. I've removed the temporary file and I've changed the name to assisted-service-leader-election-role in the original.